src/test/java/com/triviamachine/triviamachine/DatabaseEntityTest.java

```diff
@@ -0,0 +1,113 @@
+package com.triviamachine.triviamachine;
+
+import com.triviamachine.triviamachine.database.*;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import static junit.framework.TestCase.assertTrue;
+import static org.junit.Assert.*;
+
+@RunWith(SpringRunner.class)
```

Comment:
You could have just declared a "new" dateformat for all of these tests to avoid loading up the SpringRunner.

_Excellent_ test coverage, though.